db/db.py

```diff
@@ -0,0 +1,28 @@
+from dotenv import load_dotenv
+import psycopg2
+import os
+
+load_dotenv()
+
+PGUSER = os.getenv("PGUSER")
```

Comment:
it's fine to have these as environment variables for now, but when you actually write a piece of software that will use this file, we should take these as parameters.